backend/db/models/evaluation/experiment.py

```diff
@@ -94,14 +94,25 @@ class ExperimentSampleEntity(SQLModel, table=True):
         default="",
         description="Trace ID for the request"
     )
+    trace: Optional[List[dict]] = Field(
```

Comment:
Maybe just a dict is more general in this case, if we need more information about trace other than just spans.